test/utils/image/manifest.go

```diff
@@ -22,7 +22,7 @@ import (
 )
 
 const (
-	dockerHubRegistry = "docker.io"
+	dockerHubRegistry = "docker.io/library"
```

Comment:
This is not true only for the images which are present in docker library, so the library should come in imagename and not in dockerHubRegistry registry.

Reply:
The nginx images are the only use of this const at the moment. Is it necessary to draw that distinction? The goal here is to handle docker and containerd’s normalization.

Reply:
@cblecker 3 of them, not just 2.

- community docker
- docker in RHEL
- containerd

Reply:
@mkumatag Please take a look at all registries below, all of them include the full "namespace":
```
	e2eRegistry       = "gcr.io/kubernetes-e2e-test-images"
	gcRegistry        = "k8s.gcr.io"
	PrivateRegistry   = "gcr.io/k8s-authenticated-test"
 	sampleRegistry    = "gcr.io/google-samples"
```

We can change `dockerHubRegistry` to `dockerLibraryRegistry` if that is preferred.

Reply:
I'm okay with renaming to `dockerLibraryRegistry`